CMakeLists.txt

```diff
@@ -83,6 +87,12 @@ if(BUILD_EXAMPLES)
     add_subdirectory(tools)
 endif()
 
+if(BUILD_IPDEV)
+    add_subdirectory(src/ethernet)
+endif()
+
+add_subdirectory(src/compression)
```

Comment:
consider moving under src/ethernet?

Reply:
Don't we want to make COMPRESSION factory available LRS wide?

Reply:
Pending